lib/ssl/src/ssl.erl

```diff
@@ -2873,8 +2873,9 @@ add_filter(Filter, Filters) ->
 maybe_client_warn_no_verify(#{verify := verify_none,
                              warn_verify_none := true,
                              log_level := LogLevel}, client) ->
-            ssl_logger:log(warning, LogLevel, #{description => "Authenticity is not established by certificate path validation",
-                                                reason => "Option {verify, verify_peer} and cacertfile/cacerts is missing"}, ?LOCATION);
+    ssl_logger:log(warning, LogLevel, #{description => "Authenticity is not established by certificate path validation",
+                                        reason => "Option {verify, verify_peer} together with one of the options cacertfile or "
+                                        "cacerts is needed to accomplish this"}, ?LOCATION);
```

Comment:
Proposal, to clarify that authenticity isn't verified, since the text now may suggest that it isn't verified by certificate path validation, but maybe some other way, who knows...

Description: Server authenticity is unverified since certificate path validation is not enabled
Reason: Option {verify, verify_peer}, and one of 'cacertfile' or 'cacerts', are required to enable this

Reply:
Thank you for the suggestion, pushed a new version.